gh-pages/benchmarks/descriptions/stencil_2d.md

```
@@ -0,0 +1,196 @@
+- **Efficiency**: Percentage of peak memory bandwidth achieved
+
+## Typical Results
+
+| Size (W×H) | Cells   | Time (ms) | M cells/s | GB/s  |
+|------------|---------|-----------|-----------|-------|
+| 256×256    | 65K     | 0.02      | 3,250     | 78    |
+| 512×512    | 262K    | 0.07      | 3,743     | 90    |
+| 1024×1024  | 1.05M   | 0.28      | 3,750     | 90    |
+| 2048×2048  | 4.19M   | 1.12      | 3,741     | 90    |
+
+#### Intel Arc A770 GPU (OpenCL backend)
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)